Makefile

```diff
@@ -5,11 +5,15 @@ all: generate format
 generate:
 	@mkdir -p ./pb
 	@protoc \
-		--proto_path . \
+		--proto_path ./ \
 		--proto_path ./third_party \
 		--go_out=paths=source_relative:./pb \
-		./fig/tendermint/*/*/*.proto
-
+		./fig/tendermint/codec/v1/codec.proto
+	@protoc \
+		--proto_path ./ \
+		--proto_path ./third_party \
+		--go_out=paths=source_relative:./pb \
+		./fig/tendermint/transform/v1/transform.proto
```

Comment:
Keeping `./fig/tendermint/*/*/*.proto` would handle both directories.

Reply:
right. I'll change it back.